Dockerfile

```diff
@@ -13,49 +15,63 @@ RUN mamba create -qy -p /usr/local \
-
-# GitHub repo link
-ARG ASCAT="VanLoo-lab/ascat/ASCAT@v${ASCAT_VERSION}"
-ARG COPYNUMBER="igordot/copynumber@${COPYNUMBER_VERSION}"
-ARG BATTENBERG="Wedge-lab/battenberg@v${BATTENBERG_VERSION}"
-
-# Install Package Dependency toolkit
-RUN R -e 'install.packages(c("argparse", "BiocManager", "pkgdepends", "optparse"))' && \
-    R -q -e 'BiocManager::install(c("ellipsis", "splines", "VariantAnnotation"))'
-
-# Install Battenberg
-COPY installer.R /usr/local/bin/installer.R
-RUN chmod +x /usr/local/bin/installer.R
-
-RUN Rscript /usr/local/bin/installer.R -d ${COPYNUMBER} ${ASCAT} ${BATTENBERG}
-
-# Modify paths to reference files
-COPY modify_reference_path.sh /usr/local/bin/modify_reference_path.sh
-RUN chmod +x /usr/local/bin/modify_reference_path.sh && \
-    bash /usr/local/bin/modify_reference_path.sh /usr/local/lib/R/site-library/Battenberg/example/battenberg_wgs.R /usr/local/bin/battenberg_wgs.R
-
-RUN ln -sf /usr/local/lib/R/site-library/Battenberg/example/filter_sv_brass.R /usr/local/bin/filter_sv_brass.R && \
-    ln -sf /usr/local/lib/R/site-library/Battenberg/example/battenberg_cleanup.sh /usr/local/bin/battenberg_cleanup.sh
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    libxml2 \
+    libxml2-dev \
+    libcurl4-gnutls-dev \
+    build-essential \
+    libfontconfig1-dev \
+    libharfbuzz-dev \
+    libfribidi-dev \
+    libfreetype6-dev \
+    libpng-dev \
+    libtiff5-dev \
+    libjpeg-dev \
+    r-cran-rgl \
+    git \
+    libssl-dev \
+    r-cran-curl \
+    r-cran-devtools \
+    wget && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN R -q -e 'install.packages("renv")' && \
+    mkdir -p /usr/local/src
+
+COPY renv.lock /usr/local/renv.lock
+COPY battenberg_bl_custom.R /usr/local/src/
+COPY battenberg_wgs_bl_custom.R /usr/local/src/
+
+ARG ASCAT_VERSION
+ARG BATTENBERG_VERSION
+
+RUN set -eux && \
+    # Ignore specific packages from `renv.lock` file
+    R -q -e 'renv::settings$ignored.packages(c("ASCAT", "Battenberg"))' && \
+    R -q -e 'renv::restore(lockfile = "/usr/local/renv.lock")' && \
+    # Install ASCAT
+    cd /usr/local/src/ && \
+    wget -q -O ascat-${ASCAT_VERSION}.tar.gz \
+        https://github.com/VanLoo-lab/ascat/archive/refs/tags/v${ASCAT_VERSION}.tar.gz && \
+    tar -xzf ascat-${ASCAT_VERSION}.tar.gz && \
+    R CMD INSTALL ascat-${ASCAT_VERSION}/ASCAT/ && \
+    # Instal Battenberg
+    wget -q -O battenberg-${BATTENBERG_VERSION}.tar.gz \
+        https://github.com/Wedge-lab/battenberg/archive/refs/tags/v${BATTENBERG_VERSION}.tar.gz && \
+    tar -xzf battenberg-${BATTENBERG_VERSION}.tar.gz && \
+    cp battenberg_bl_custom.R battenberg-${BATTENBERG_VERSION}/R/battenberg.R && \
+    cp battenberg_wgs_bl_custom.R battenberg-${BATTENBERG_VERSION}/inst/example/battenberg_wgs.R && \
+    R CMD INSTALL battenberg-${BATTENBERG_VERSION}/ && \
+    # Cleanup
+    cd /usr/local/ && \
+    rm -rf /usr/local/src
```

Comment:
**suggestion**
```suggestion
    rm -rf /usr/local/src/*
```
/usr/local/src is typically present in the OS so we don't want to delete that level of the directory